docs/pages/web/badge.js

```diff
@@ -6,7 +6,6 @@ import docgen, { type DocGen } from '../../docs-components/docgen.js';
 import PageHeader from '../../docs-components/PageHeader.js';
 import MainSection from '../../docs-components/MainSection.js';
 import QualityChecklist from '../../docs-components/QualityChecklist.js';
-
```

Comment:
unrelated change, should be reverted for a cleaner PR

Reply:
Right, it's cleanup that can be addressed the next time someone does work on this file. As it stands, anyone looking into the git history of this file will see this PR, adding noise to their search as this PR has no material changes to the file.